### PR TITLE
docs: update project ideas

### PR DESCRIPTION
**Pull request by magnus-madsen**

### CODE REVIEW

File: docs/PROJECTS.md [hunk 1]
```diff
@@ -4,7 +4,63 @@ Here is a list of projects that are currently open for exploration.
...
+
+## Compiler Fuzzing
+
+Compilers are large and complex pieces of software. The correctness of a
+compiler is paramount: A compiler that silent mis-compiles (i.e. wrongly
+translates) a program is dangerous: We cannot trust the programs we compile and
+run! We can test compilers by writing unit tests, but unfortunately such tests
+tend to only test the 'happy path' of the compiler. Moreover, the amount of unit
+tests that can be written is limited. Instead, compiler fuzzing techniques have
+been proposed. A compiler fuzzy typically takes a test suite as input and subtly
```
Comment (mlutze) [inline docs/PROJECTS.md:54]: fuzzer

File: docs/PROJECTS.md [hunk 2]
```diff
@@ -4,7 +4,63 @@ Here is a list of projects that are currently open for exploration.
...
+package manager for the Flix programming language. The work will include reading
+papers, language design, and implementation in a real-world programming language.
+
+## Compiler Fuzzing
+
+Compilers are large and complex pieces of software. The correctness of a
+compiler is paramount: A compiler that silent mis-compiles (i.e. wrongly
+translates) a program is dangerous: We cannot trust the programs we compile and
+run! We can test compilers by writing unit tests, but unfortunately such tests
+tend to only test the 'happy path' of the compiler. Moreover, the amount of unit
```
Comment (mlutze) [inline docs/PROJECTS.md:52]: number

File: docs/PROJECTS.md [hunk 3]
```diff
@@ -4,7 +4,63 @@ Here is a list of projects that are currently open for exploration.
...
+particular input, the framework automatically simplifies the input to find the
+simplest case that causes the failure. This helps developers identify and fix
+bugs in their code efficiently.
+
+The aim of this project is to (1) explore the design space of QuickCheck
+frameworks and to (2) design and implement a QuickChecker for the Flix
+programming language. The work will include reading papers, language design, and
+implementation in a real-world programming language.
+
+# Package Management
```
Comment (KengoTODA) [inline docs/PROJECTS.md:30]: we probably miss one `#` here:

```suggestion
## Package Management
```

**Comment by magnus-madsen** [comment]
@mlutze Can you approve, so I can merge?

**Comment by magnus-madsen** [comment]
or maybe @JonathanStarup  :)

**Review by JonathanStarup — APPROVED** [review]

**Review by mlutze — APPROVED** [review]

**Pull request merged**